deno_typescript/lib.rs

```diff
@@ -127,6 +127,7 @@ pub fn compile_bundle(
 
   let config_json = serde_json::json!({
     "compilerOptions": {
+      "strict": true,
```

Comment:
DOH, I didn't realise that we had eliminated this from here, and that our internal code had drifted so far... 😭 sorry...